src/PayloadListener/Services/PayloadListenerService.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Monai.Deploy.WorkflowManager.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+
+namespace Monai.Deploy.WorkflowManager.PayloadListener.Services
+{
+    public class PayloadListenerService : ListenerServiceBase
```

Comment:
@jackschofield23 are we going to have a different service for each queue we're listening to?

Reply:
The base class makes sense to me, it's the subclass that I'm confused about. If you not instantiating the subclass, why have it at all? Is inheritance the best pattern here?

Reply:
Thats what I was thinking, I originally based it off of how it was done in the informatics gateway, but it may be worth ditching the inheritance, Im happy to have a crack at that.